Sources/LightweightCharts/LightweightChartsModels/MouseEventParams.swift

```diff
@@ -4,50 +4,135 @@ import Foundation
 public enum EventTime {
     case utc(timestamp: Double)
     case businessDay(BusinessDay)
+    case businessDayString(String)
 }
 
 // MARK: -
 public enum EventPrices {
-    case barPrice(BarPrice)
-    case barPrices(BarPrices)
+    case barData(BarData)
+    case lineData(LineData)
     case none
 }
 
 // MARK: -
-public struct BarPrices: Codable {
```

Comment:
That breaking change does not comply with Android and Js libs

Reply:
https://github.com/search?q=repo%3Atradingview%2Flightweight-charts%20BarPrices&type=code

Reply:
https://github.com/search?q=repo%3Atradingview%2Flightweight-charts-android%20BarPrices&type=code

Reply:
both of js types barData and barPrice have same signature. We can't separate these types without additional meta data.